docker/conda/environments/requirements.txt

```diff
@@ -6,6 +6,7 @@
 # Packages listed here should also be listed in setup.py
 git+https://github.com/nv-morpheus/dfencoder.git@branch-22.09#egg=dfencoder
 nvidia-pyindex
+pytest-benchmark>=4.0
```

Comment:
Why is this here instead of in the conda environment?

Reply:
I originally added it in the yaml, but conda would report that nothing provides the package. Putting it in the requirements worked.

Reply:
At the time yesterday pytest-benchmark 4.0 hadn't been uploaded to conda-forge, it is now, so I could move it back to the yaml.